includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -99,4 +99,40 @@ private function get_learner_courses_page_content() {
 		return implode( $blocks );
 	}
 
+	/**
+	 * Get the block content for course completed.
+	 *
+	 * @return string
+	 */
+	private function get_course_completed_page_content() {
+		$blocks   = [];
+		$blocks[] = serialize_block(
+			[
+				'blockName'    => 'core/paragraph',
+				'innerContent' => [ '<p class="has-text-align-center">' . __( 'Congratulations on completing the course!', 'sensei-lms' ) . '</p>' ],
+				'attrs'        => [ 'align' => 'center' ],
+			]
+		);
+
+		$blocks[] = serialize_block(
+			[
+				'blockName'    => 'sensei-lms/course-results',
+				'innerContent' => [],
+				'attrs'        => [],
+			]
+		);
+
+		$find_my_courses_label = __( 'Find More Courses', 'sensei-lms' );
+
+		$blocks[] = serialize_block(
+			[
+				'blockName'    => 'core/buttons',
```

Comment:
This should be the Course Completed Actions block.

Reply:
But I think this means the "View Certificate" button won't ever be added, is that right?